packages/turbo/ios/RNVisitableView.swift

```diff
@@ -19,6 +19,7 @@ class RNVisitableView: UIView, RNSessionSubscriber {
   }
   @objc var onMessage: RCTDirectEventBlock?
   @objc var onVisitProposal: RCTDirectEventBlock?
+  @objc var onOpenExternalUrl: RCTDirectEventBlock?
```

Comment:
Renaming this to `onOpenExternalURL` would cause the app to crash :D We won't loose much if we just avoid upper-case url word

Reply:
Haha, I'm good with both. No strong opinions here, so let's keep it lowercase.